components/badge.css

```diff
@@ -0,0 +1,78 @@
+@import url(/stylesheets/main.css);
+
+.badge {
+  width: 0.8rem;
+  height: 0.8rem;
+  padding: 1px;
+  font-size: 12px;
+  border-radius: 50%;
+  background-color: hsl(var(--secondary-color));
+  text-align: center;
+  color: white;
+}
+
+.align-badge {
+  position: relative;
+  top: 1.5rem;
+  left: 2.8rem;
+}
+
+.badge-s {
+  position: relative;
+  bottom: 3rem;
+  left: 2.5rem;
+}
+
+.badge-m {
```

Comment:
I'm unable to figure out why all the badge classes that are the children themselves are given relative positioning. Maybe there should be absolute positioning. You might want to check that.

Reply:
the reason for positioning relative was to adjust their position w.r.t the avatar and icon on which they positioned.
You can suggest some alternate approach if you have in mind.